Patches/EPOE Forked/Bionics_Patch.xml

```diff
@@ -0,0 +1,266 @@
+						<ShootingAccuracyPawn>+1</ShootingAccuracyPawn>
+						<AimingAccuracy>0.5</AimingAccuracy>
+						<AimingDelayFactor>-0.15</AimingDelayFactor>
+						<MentalBreakThreshold>-0.06</MentalBreakThreshold>
+						<Suppressability>-0.50</Suppressability>
+					</statOffsets>
+				</value>
+			</li>
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/HediffDef[defName="EPIA_AuxiliaryAI_Brawler"]/stages/li/statOffsets</xpath>
+				<value>
+					<statOffsets>
+						<ShootingAccuracyPawn>-1.2</ShootingAccuracyPawn>
+						<AimingAccuracy>-0.6</AimingAccuracy>
+						<MentalBreakThreshold>-0.14</MentalBreakThreshold>
+						<MeleeHitChance>8</MeleeHitChance>
+						<MeleeCritChance>0.50</MeleeCritChance>
+						<MeleeParryChance>0.50</MeleeParryChance>
+						<MeleeDodgeChance>0.50</MeleeDodgeChance>
+						<Suppressability>-1</Suppressability>
+					</statOffsets>
+				</value>
+			</li>
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/HediffDef[defName="EPOE_InstinctOptimizedEyes"]/stages/li[minSeverity="0.99"]/statOffsets/MeleeDodgeChance</xpath>
+				<value>
+					<MeleeDodgeChance>0.05</MeleeDodgeChance>
+				</value>
+			</li>
+			
+			<li Class="PatchOperationConditional">
+				<xpath>Defs/HediffDef[defName="EPOE_InstinctOptimizedEyes"]/stages/li[minSeverity="0.99"]/statOffsets/MeleeCritChance</xpath>
+				<nomatch Class="PatchOperationAdd">
+					<xpath>Defs/HediffDef[defName="EPOE_InstinctOptimizedEyes"]/stages/li[minSeverity="0.99"]/statOffsets</xpath>
+					<value>
+						<MeleeCritChance>0.075</MeleeCritChance>
+						<MeleeParryChance>0.075</MeleeParryChance>
+					</value>
+				</nomatch>
+				<match Class="PatchOperationAdd">
+					<xpath>Defs/HediffDef[defName="EPOE_InstinctOptimizedEyes"]/stages/li[minSeverity="0.99"]/statOffsets</xpath>
+					<value>
+					</value>
+				</match>
+			</li>
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/HediffDef[defName="EPOE_InstinctOptimizedEyes"]/stages/li[label="Optimizing"]/statOffsets/ShootingAccuracyPawn</xpath>
+				<value>
+					<AimingAccuracy>0.1</AimingAccuracy>
+				</value>
+			</li>
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/HediffDef[defName="EPOE_InstinctOptimizedEyes"]/stages/li[minSeverity="0.99"]/statOffsets/ShootingAccuracyPawn</xpath>
+				<value>
+					<AimingAccuracy>0.25</AimingAccuracy>
+				</value>
+			</li>
+			<li Class="PatchOperationConditional">
+				<xpath>Defs/HediffDef[defName="EPOE_InstinctOptimizedHand"]/stages/li[label="Optimizing"]/statOffsets/MeleeCritChance</xpath>
+				<nomatch Class="PatchOperationAdd">
+					<xpath>Defs/HediffDef[defName="EPOE_InstinctOptimizedHand"]/stages/li[label="Optimizing"]/statOffsets</xpath>
+					<value>
+						<MeleeCritChance>0.15</MeleeCritChance>
+						<MeleeParryChance>0.15</MeleeParryChance>
+					</value>
+				</nomatch>
+				<match Class="PatchOperationAdd">
+					<xpath>Defs/HediffDef[defName="EPOE_InstinctOptimizedHand"]/stages/li[label="Optimizing"]/statOffsets</xpath>
```

Comment:
An empty Add value here?

Reply:
Not having a match in the test class seems to cause it to throw an error. The match here is to check whether the EPOE forked intergrated patch has already added the stat so it doesn't do a duplicate stat in the bionic.